XIVComboVX/Combos/WHM.cs

```diff
@@ -5,8 +5,7 @@ namespace PrincessRTFM.XIVComboVX.Combos;
 internal static class WHM {
 	public const byte JobID = 24;
 
-	public const uint
-		Stone = 119,
+	public const uint Stone = 119,
```

Comment:
For consistency, please make sure that the multi-line declarations for action IDs, debuff IDs, and level thresholds all conform to the rest of the project: `public const <type>` on one line, then each declaration on a separate line below, indented one additional level.